File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    s_spt = s.split(" ")
    let res = []
    
    s_spt.map((v, i) => {
        temp = v[0].toUpperCase() + v.slice(1).toLowerCase()
        res.push(temp)
    })

    
    return(res.join(" "))    
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
import java.util.*;

class Solution {
    public String solution(String s) {
        StringBuilder sb = new StringBuilder();
        char[] sArr = s.toCharArray();
        boolean isFirst = true;
        for (char c: sArr) {
            if (c == ' ') {
                sb.append(c);
                isFirst = true;
                continue;
            }
            if (isFirst) {
                sb.append(Character.toUpperCase(c));
                isFirst = false;
                continue;
            }
            sb.append(Character.toLowerCase(c));
        }
        
        return sb.toString();
    }
}